### DISCUSSION TITLE
Unable to reverify github students developer pack because of email.

### DISCUSSION THREAD

**Discussion by SciSaif** [community/community#50779] [GitHub Education]
I am unable to reverify my github students developer pack because of this: 
![image](https://user-images.githubusercontent.com/51053279/227109538-cd000291-921b-4cec-bd5a-a45b87e0735a.png)
It's asking me to use the email provided by the college, but our college rarely gives out emails. Most of the students have not been provided with a college email. When we ask them for it, they say its not possible. 
What should we do in such cases? 
It was working fine for 2 years, but this year they require college email. 
Can someone please help us?

**Comment by rLakshmeesh** [comment]
You can use the college id card to get student developer pack.

**Reply by SciSaif** [reply]
> You can use the college id card to get student developer pack.

Yes but it doesnt let me proceed without selecting a college email id

**Comment by rLakshmeesh** [comment]
just type your college name don't select the suggestion from the github.It
will allow to continue and give your location and give the id card of your
college( just take pic of both side).

**Reply by SciSaif** [reply]
> just type your college name don't select the suggestion from the github.It
> will allow to continue and give your location and give the id card of your
> college( just take pic of both side).
> 
> On Thu, Mar 23, 2023 at 12:28 PM saifullah rahman ***@***.***>
> wrote:
> 
> > You can use the college id card to get student developer pack.
> >
> > Yes but it doesnt let me proceed without selecting a college email id
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/orgs/community/discussions/50779#discussioncomment-5401371>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/A4ARUUGGUV4R5S7CSGXKWNLW5PYBBANCNFSM6AAAAAAWEWT5CI>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
> 

It was working yesterday but now it won't let me proceed, it says: We require applicants to use a school-issued email address to apply

**Comment by AzizBenIsmail** [comment]
You can use Ur student card

**Reply by SciSaif** [reply]
I am not able to proceed without adding a college email id

**Comment by Aditya753b** [comment]
yeah me too same issue please anyone help me to renew it

**Reply by ZapeeoSheikh** [reply]
Facing the same issue

**Reply by Aditya753b** [reply]
hey mine is done... can i help you?

**Reply by BlexDevop** [reply]
how did you make it work my document is always rejected although a year ago it worked with the document now i get this answer "Attendees of this school must verify and use their academic email address".
but we do not get our own email from our school we use our private email address to communicate with the teachers

**Reply by Aditya753b** [reply]
well most of the rejections are faced due to not having an .edu mail. Even though github says there is an alternative way to get the apllication accepted by giving correct documents but actually u need to have an edu mail to make it work. Does ur staff/teachers have edu mail?

**Comment by D7x7z49** [comment]
We should gather together. Make them aware of this issue. 👉 <https://github.com/orgs/community/discussions/59360>

Let the problem be taken seriously. 💔

**Comment by Aditya753b** [comment]
> how did you make it work my document is always rejected although a year ago it worked with the document now i get this answer "Attendees of this school must verify and use their academic email address". but we do not get our own email from our school we use our private email address to communicate with the teachers



> I am not able to proceed without adding a college email id

is it working now?

**Comment by Ajay-singh1** [comment]
Yeah My College Is Also Not Providing My College E-Mail I'd What to do?

**Reply by Aditya753b** [reply]
is it providing for staff?
